### DISCUSSION TITLE
订阅链接的剩余流量的从哪里读的数据

### DISCUSSION THREAD

**Discussion by mambalong** [clash-verge-rev/clash-verge-rev#1698] [Q&A] [answered]
好奇订阅链接的剩余流量的从哪里读的数据，为什么配置文件里没有？
![image](https://github.com/user-attachments/assets/0ae9c5ef-0f2f-4514-bc16-d70a013b458f)

**Comment by HozumiMoa** [comment, marked as answer]
https://www.clashverge.dev/guide/url_schemes.html
